shared/bundle_analysis/models.py

```diff
@@ -43,11 +43,12 @@
 
 create table chunks (
     id integer primary key,
+    session_id integer not null,
     external_id text not null,
     unique_external_id text not null,
     entry boolean not null,
     initial boolean not null,
-    unique (unique_external_id)
```

Comment:
Lifting this uniqueness constraint. Discussed with BA folks, that this shouldn't be here to begin with.